tests/Feature/AdjustmentTest.php

```diff
@@ -34,6 +34,25 @@ public function test_can_adjust_wallet_balance($initial_balance, $new_balance)
             ]);
     }
 
+    /**
+     * @dataProvider walletBalanceProvider
+     */
+    public function test_unauthorized_user_cannot_adjust_wallet_balance($initial_balance, $new_balance)
```

Comment:
the test is perfect, but I think we don't need to use the data provider, the balance amount is not related to the authorization check